src/OrchardCore.Modules/OrchardCore.Sms/Views/TwilioSettings.Edit.cshtml

```diff
@@ -0,0 +1,51 @@
+@using OrchardCore.Sms.ViewModels
+@using OrchardCore.Sms
+
+@model TwilioSettingsViewModel
+
+<div class="sms-settings-container d-none" id="SmsSettingsContainer@(SmsConstants.TwilioServiceName)">
+    <h4>@T["Twilio Account Info"]</h4>
+    <div class="mb-3" asp-validation-class-for="PhoneNumber">
+        <label asp-for="PhoneNumber">@T["Phone Number"]</label>
+        <input asp-for="PhoneNumber" class="form-control" />
+        <span asp-validation-for="PhoneNumber"></span>
+        <span>@T["Phone number must include a country code. For example, +1 for United States."]</span>
```

Comment:
Please use `class="hint"` in help messages.

Reply:
@agriffard done!